### PR TITLE
Added travis-ci and fixed coding standards issues

### PR DESCRIPTION
**Pull request by christianselig**
Added [travis-ci](http://travis-ci.org) to the module and fixed coding standards issues.

### CODE REVIEW

**Comment by christianselig** [comment]
Should be working, tested travis with Dan's scripts and it's working with them (exits with 0).

File: theme/theme.inc.orig [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<?php
```
Comment (jordandukart) [inline theme/theme.inc.orig:1]: Merge conflicts?

File: tests/islandora_pdf_ingest_purge.test.orig [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+    // Checks the FULL_TEXT datastream for correct text.
+    if (isset($object['FULL_TEXT'])) {
+      $this->drupalGet('islandora/object/' . $object . '/datastream/FULL_TEXT/view');
+      $this->assertText('Misthallery, Gressenheller, Folsense, Akbadain', 'Text from tests/fixtures/test.txt found in the FULL_TEXT datastream', 'Islandora');
+    }
+
+    // Attempts to delete the object and then collection.
+    $this->deleteObject($object);
+>>>>>>> df8eb5fea744dfce8d31aecc077502664fe3c5b3
```
Comment (jordandukart) [inline tests/islandora_pdf_ingest_purge.test.orig:162]: Merge conflicts?

File: tests/islandora_pdf_ingest_purge.test.orig [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+    $edit = array(
+      'imagemagick_convert' => $this->imagemagick_convert,
+    );
+    $this->drupalPost('admin/config/media/image-toolkit', $edit, 'Save configuration');
+
+    // Attempts to create an object.
+    $path = 'islandora/object/' . $collectionpid . '/manage/overview/ingest';
+    $file = drupal_get_path('module', 'islandora_pdf') . '/tests/fixtures/test.pdf';
+    $text = drupal_get_path('module', 'islandora_pdf') . '/tests/fixtures/test.txt';
+<<<<<<< HEAD
```
Comment (jordandukart) [inline tests/islandora_pdf_ingest_purge.test.orig:78]: So many merge conflicts.

**Comment by jordandukart** [comment]
This pull should probably be closed. For the Travis standards forking off the main repo and codesniffering what needs to be done would probably be the best bet. Dan has a separate pull request for his tests.

**Pull request closed without merging**